doc/latex.rst

```diff
@@ -226,6 +226,21 @@ into the generated ``.tex`` files.  Its ``'sphinxsetup'`` key is described
 
      .. versionadded:: 1.5
 
+  ``'packages'``
+     List of additional LaTeX packages which is a pair of package name and
+     package options.  For example:
+
+     .. code-block:: python
+
+         latex_elements = {
+             'packages': [('isodate', 'english'),  # => \usepackage[english]{isodate}
+                          ('xcomment', None)],     # => \usepackage{xcomment}
+         }
+
+     It defaults to empty.
```

Comment:
Perhaps say the package loading is inserted after Sphinx own LaTeX style file and all packages it loads except for `hyperref` and  packages loaded from Sphinx extensions? and that packages which need to be loaded after the latter should be inserted via ``'preamble'`` key?